### ISSUE TITLE
Add timer and elapsed time to checks

### ISSUE THREAD

**Issue by paurullan** [codingjoe/django-health-check#123]
We are adopting django-health-check as our base health check helper and one thing we had in our custom health checks was showing the elapsed time. For example:

```python
from contexttimer import Timer
from api.models import City

def health_check_view(request):
    with Timer() as timer_db:
        City.objects.first()
    response = "ok; db: {}".format(timer_db.elapsed)
    return HttpResponse(response)
```

I am planning to put this in django-health-check but you do not have any package dependency. Will you accept the feature even if it forces a dependency?
https://pypi.python.org/pypi/contexttimer/0.3.3

**Comment by codingjoe** [comment]
Hi @paurullan,

sure if it makes sense. But since the method `check` is always called, you could just take the time before and after using `time.time`. That way you wouldn't need to add the dependency.

I'm really looking forward to see you contribution. I think it's a really cool feature.

Cheers
-Joe

**Comment by carderm** [comment]
Hi, I've added a pull request to include a simpler version of this just using the below:
from timeit import default_timer as timer
which doesn't require any other apps -

**Comment by codingjoe** [comment]
The timer was merged in 5ec5873aea4410d43b299aeda3dd3b3ceb13657a and released in v2.4.0 👍 

Thanks everyone, great work 🥇
